clients/src/main/resources/common/message/OffsetFetchRequest.json

```diff
@@ -39,6 +41,8 @@
         "about": "The topic name."},
       { "name": "PartitionIndexes", "type": "[]int32", "versions": "0+",
         "about": "The partition indexes we would like to fetch offsets for." }
-    ]}
+    ]},
+    {"name": "RequireStable", "type": "bool", "versions": "7+", "default": "false",
```

Comment:
It was my suggestion, but maybe "stable" is not the best term. How about `requireLatest` or `requireCurrent`?

Reply:
I don't see a real difference, stable is actually better to indicate a `real committed` state